10x/count/count.wdl

```diff
@@ -396,6 +405,8 @@ task align_reads_split {
   File reference_path
   
   command <<<
+    # todo is this necessary here? is the reference itself used or just the path to it?
+    mkdir genome_reference && tar -xf ${reference_path} -C genome_reference --strip-components 1
```

Comment:
I believe all it's using is the reference.json in the reference path. I think there's going to be a bunch of optimizations like this where assumptions about a shared filesystem lead to inefficiencies when running on GCE. Probably shouldn't hold up merging this, but we should make a note for a future optimization ticket.

Reply:
Great comment, and I agree with the sentiment. We shouldn't fix it now, but I made a ticket!